paddle/function/nnpack/nnpack.cmake

```diff
@@ -0,0 +1,16 @@
+# Find the NNPACK library
```

Comment:
Thank you for your comments. I also think automatically download the source code and compile it is better.